stripe/src/resources/card.rs

```diff
@@ -1,30 +1,22 @@
+use std::collections::HashMap;
+
+use super::currency::Currency;
```

Comment:
Other files follow the pattern `use resources::Currency;`, so lets do that for consistency.

(Until we decide to upgrade to rust2018 😆 )